.evergreen/run-mockupdb-tests.sh

```diff
@@ -2,19 +2,17 @@
 
 set -o xtrace
 set -o errexit
-
+cd $PROJECT_DIRECTORY
```

Comment:
Ideally we can run this script locally with no setup. Let's define `PROJECT_DIRECTORY` ourselves:
```
PROJECT_DIRECTORY=$(dirname $(dirname $0))
```

Reply:
Done.